openapi.yaml

```diff
@@ -463,6 +464,61 @@ paths:
             <repeatMin>0</repeatMin>
         </ns2:message>
         ```
+        * * *
+
+        ### Opt-out and Unsubscribe
+
+        Whispir provides a way for clients who require sending to a marketing email to include two new headers as directed by mail inboxes providers.
+        It creates a way for recipients to opt out of email lists easily, which will add up to a sender reputation and deliverability.
+
+        This additional request body is optional under the `email` property.
+
+        ```JSON
+        POST https://api.<region>.whispir.com/workspaces/{workspaceId}/messages
+         
+        Authorization: Basic am9obi5zbWl0aDpteXBhc3N3b3Jk
+        x-api-key: your_api_key
+        Content-Type: application/vnd.whispir.message-v1+json
+        Accept: application/vnd.whispir.message-v1+json
+        In this sample we will send a message with an opt out option property
+        {
+            "to" : "jsmith@test.com",
+            "subject" : "Test Opt-out and Unsubscribe",
+            "email" : {
+                "body" : "This is a test message",
+                "type" : "text/plain"
+                "optOutOption": {
+                  "rmsUnsubscribeUrl": "@@web_link@@#unsubscribe",
+                  "oneClickApiMapping": "unsubscribe"
```

Comment:
can you please update the oneClickApiMapping value to a sample generated api mapping id, something like `2ab64862`

Reply:
I'll resolve this comment :) @lnachor-whispir